app/helpers/ems_cluster_helper/textual_summary.rb

```diff
@@ -150,6 +150,8 @@ def textual_allvms_size
   end
 
   def textual_total_miq_templates
+    return nil if @record.kind_of?(EmsClusterOpenstackInfra)
```

Comment:
It was agreed as an acceptable way for now https://github.com/ManageIQ/manageiq/pull/2720

@martinpovolny has in mind enhancements, that will make this better in the future

I do it by respond_to in some places, but that works only for new methods specific for OpenStackInfra.

The question is, if we want to hide this simply, before we design better solution. Not sure how long it will take to redesign the textual pages?